executor/builder.go

```diff
@@ -330,7 +330,7 @@ func (b *executorBuilder) buildShowSlow(v *plannercore.ShowSlow) Executor {
 }
 
 // buildIndexLookUpChecker builds check information to IndexLookUpReader.
```

Comment:
I think only renaming the struct is not enough. 

For example, the comment of this function mentions `IndexLookUpReader` but the parameter is `PhysicalIndexLoopUpGather`. IMO, this will confuse new contributors.